x/auth/vesting/alias.go

```diff
@@ -0,0 +1,35 @@
+// nolint
+// autogenerated code using github.com/rigelrozanski/multitool
+// aliases generated for the following subdirectories:
+// ALIASGEN: github.com/cosmos/cosmos-sdk/x/auth/vesting/types/
+package vesting
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
```

Comment:
exported shouldn't be aliased

Reply:
FYI, I fixed this in `auth/alias.go` as well.